oshdb/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+Changelog OSHDB MODULE
+======================
+
+## 0.7
+
+### renaming
+ - OSMWay.getRef() -> OSMWay.getMember()
+ - OSHDBTimestamp.getRawUnixTimestamp() -> OSHDBTimestamp.getEpochSecond()
+ 
+### moving
+ - oshdb.util.OSHDBTimestamp - oshdb.OSHDBTimestamp
+ - oshdb.util.OSHDBTag - oshdb.OSHDBTag
+ - CellIterator.OSHEntityFilter/OSMEntityFilter - oshdb.osh.OSHEntityFilter, oshdb.osm.OSMEntityFilter
+ - oshdb-api.generic.function - oshdb.util.function
+ 
+
+### breaking changes
+
+### new features
+ - new interface OSHDBTimeable
+ - new interface OSHDBBoundable
+
```

Comment:
ehrm. not sure if I like the separate per-module changelog. Why not just include this in the general CHANGELOG (perhaps slightly condensed)? Especially since this includes breaking changes (btw, aren't moved and renamed classes/methods are **breaking changes** since people do need to adjust their code accordingly).

Reply:
this is done, isn't it? 
removed the changelog from oshdb module and added it to oshdb-parent changelog.

Reply:
yes!